core/src/main/java/io/fabric8/maven/core/config/PlatformMode.java

```diff
@@ -66,6 +66,10 @@ public String getLabel() {
      * Returns true if the given maven properties indicate running in OpenShift platform mode
      */
     public static boolean isOpenShiftMode(Properties properties) {
-        return Objects.equal(openshift.toString(), properties.getProperty(FABRIC8_EFFECTIVE_PLATFORM_MODE, ""));
+        if (properties == null) {
```

Comment:
Maybe I will use a ternary in this case just for make code clean, but just my opinion of course.

Reply:
Yup, good point!